src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBroker.cs

```diff
@@ -27,6 +27,9 @@ internal class iOSBroker : NSObject, IBroker
 
         private readonly ICoreLogger _logger;
         private readonly ICryptographyManager _cryptoManager;
+        private string BrokerRequestNonce { get; set; }
+        private string BrokerResponseNonce { get; set; }
+        public static bool DoesNonceMatch { get; set; } = false;
```

Comment:
There doesn't seem to be any need to have this property as public? Or even have it at all.